components/desktop/gnome2/gnome-games/Makefile

```diff
@@ -29,7 +30,8 @@ include $(WS_TOP)/make-rules/prep.mk
 include $(WS_TOP)/make-rules/configure.mk
 include $(WS_TOP)/make-rules/ips.mk
 
-PATH=/usr/gnu/bin:/usr/bin
+# Need automake 1.11
```

Comment:
Why do you add these comments? We use automake 1.15, all components which required automake 1.11, as far as I know, were patched.